src/axom/spin/Primitives.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#ifndef SPIN_PRIMITIVES_HPP_
+#define SPIN_PRIMITIVES_HPP_
+
+#include "axom/primal/geometry/BoundingBox.hpp"
+#include "axom/primal/geometry/NumericArray.hpp"
+#include "axom/primal/geometry/Point.hpp"
+#include "axom/primal/geometry/Vector.hpp"
+
+namespace axom
+{
+namespace spin
+{
+  using primal::BoundingBox;
+  using primal::NumericArray;
+  using primal::Point;
+  using primal::Vector;
```

Comment:
@agcapps , as we discussed, the fact that you are "importing" primal datatypes within the ``spin`` namespace here is perhaps the biggest argument for why the spatial acceleration data structures are tightly coupled to the geometric primitive types and therefore belong together.

I understand why you are doing it, but creating these aliases here and exposing them in application is at least confusing. For example, the application can now use ``spin::BoundingBox`` but, that class is not part of ``spin`` since it is in ``primal`` that creates confusion. 

It would be better to put all this in an ``internal`` namespace to discourage ``spin::BoundingBox`` to

Reply:
I removed the spin alias to primal types, as recommended by yourself and by @kennyweiss .